app/ds.py

```diff
@@ -20,4 +21,22 @@
 # returns the last 4 characters of the
 # generated uuid
 def get_session_id():
-    return str(uuid.uuid4())[-4:]
\ No newline at end of file
+    return str(uuid.uuid4())[-4:]
+
+def get_alias():
+    return random.choice(["John","Smith","Suzie","ShoeDog","Bob","Nicolas",
+    "Barbra","Henry","Thomas","Ronald","Margaret","Claudia","Berta","Agnes",
+    "Mary","Paula","Paul","Ivan","Alexey","Gretta","Alexandra","Svetlana",
+    "Natasha","Laura","Aarav","Vivaan","Aditya","Arjun","Shaurya","Adhira",
+    "Charu","Eka","Hiya","Keya"])
+
+def get_profession():
+    quality = random.choice(["Apprentice","Wannabe","Mature","Elder","Underqualified",
+    "Overqualified","Accomplished","Reknown","Wealthy","Famous","Avoided","Strange",
+    "Unwelcoming","Friendly","Surprising","Optimistinc","Pessimistic","Angry","Happy",
+    "Boring","Annoying","Smelly","Bright","Fun","Trendy","Upcoming"])
+    position = random.choice(["Barber","Shopkeep","Librarian","Smith","Guard",
```

Comment:
@6lor Can we split this into 2 functions? one for profession and one for quality?
The `return` syntax doesn't work for python 2.7

Reply:
Also add `pedro pedro` 😄